report.js

```diff
@@ -226,6 +242,18 @@ async function getDependencyInfo (packageJSONOverride) {
   })
 }
 
+async function getGitSha () {
+  const promise = new Promise((resolve, reject) => {
+    exec(`cd ${rootPath} && git rev-parse HEAD`, { timeout: execTimeout, maxBuffer: 1024 * 1024 * 1024 }, processGitRevParseOutput(resolve, reject))
```

Comment:
Going to the rootPath is only sometimes what we'd want. I think we should be going to the package importing scarf-js, ie the `parentPath` rather than the `rootPath`. This does handle Superset's specific use case but this will actually grab the wrong SHA in other cases.

Reply:
Oh I didn't realize that. Ok yeah then perhaps let's explicitly check for that and document that behavior in case we want to update that later.

Reply:
i at least updated the function name that this only gets the root path git sha